### PR TITLE
Solve frost-list sort overflow

### LINKED ISSUE #161: frost-list-header should have a dynamic height
Currently the frost-list-header has a static height. when we add more items like frost-sort items they seem to be cut off.

### PR DESCRIPTION
**Pull request by EWhite613**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

## Issue:
screen shot 2017-11-20 at 12 45 30 pm

## Resolution:
screen shot 2017-11-20 at 10 49 37 am

# CHANGELOG
* Fixes #161 : Have frost-list flex-wrap sort if it overflows

### CODE REVIEW

File: addon/styles/_frost-list-header.scss [hunk 1]
```diff
@@ -2,11 +2,29 @@
...
     padding-right: $frost-list-scroll-rail-gutter-width;
   }
+
+  .frost-list-header-sort-title {
+    color: $frost-color-grey-4;
+    font-size: $frost-font-s;
+    white-space: nowrap;
+  }
+
+  .frost-sort {
```
Comment (quincyle) [inline addon/styles/_frost-list-header.scss]: Should this css change go into sort?

Comment (EWhite613) [reply addon/styles/_frost-list-header.scss]: Ya, I can put it into frost-sort

**Comment by EWhite613** [comment]
Depends on: https://github.com/ciena-frost/ember-frost-sort/pull/75

File: addon/styles/_frost-list-header.scss (hunk 1 above)
Comment (EWhite613) [reply addon/styles/_frost-list-header.scss]: @quincyle  Move changes into ciena-frost/ember-frost-sort#75

**Pull request merged**